etc/di.xml

```diff
@@ -560,6 +560,7 @@
                 <item name="transaction" xsi:type="string">Adyen\Payment\Gateway\Request\CcAuthorizationDataBuilder</item>
                 <item name="vault" xsi:type="string">Adyen\Payment\Gateway\Request\VaultDataBuilder</item>
                 <item name="threeds2" xsi:type="string">Adyen\Payment\Gateway\Request\ThreeDS2DataBuilder</item>
+                <item name="redirect" xsi:type="string">Adyen\Payment\Gateway\Request\RedirectDataBuilder</item>
```

Comment:
if this includes some 3ds1 redirect logic, should this also apply to `AdyenPaymentOneclickAuthorizeRequest` for billing agreements?

Reply:
Thanks @Aleffio, indeed it is missing https://github.com/Adyen/adyen-magento2/pull/986